concepts/validator/index.md

```diff
@@ -164,66 +227,95 @@ Account addresses owning liquid newton and receiving staking reward revenue are:
 
 - Staking rewards earned from their own [self-bonded](/glossary/#self-bonded) stake.
 - Commission charged on [delegated](/glossary/#delegated) stake per the delegation rate they charge as commission.
 - The priority fee 'tip' that may be specified in a transaction and which is given to the block proposer as an incentive for including the transaction in a block.
-- [Slashing rewards](/concepts/accountability/#slashing-rewards) earned for reporting slashed faults in the [accountability and fault detection](/concepts/accountability/) protocol.
-
+- [Slashing rewards](/concepts/accountability/#slashing-rewards) earned for reporting slashed faults in the [accountability and fault detection](/concepts/accountability/) protocol. Slashing rewards are provided by forfeiture of an offending validator’s staking rewards.
 
-Staking reward revenue potential is determined by the frequency with which a validator is an active member of the consensus committee. This is driven by:
+Staking reward revenue potential is determined by the amount of stake bonded to them (their [voting power](/glossary/#voting-power)) and the frequency of their participation in the consensus committee. This is driven by:
 
 -  The amount of stake the validator has bonded to it.
 -  The committee size and number of registered validators.
 -  The frequency with which the validator proposes blocks.
 -  The amount of transaction revenue earned from transactions included in blocks committed when the validator is a member of the committee.
--  The validator's commission rate. The percentage amount deducted by a validator from staking rewards before rewards are distributed to the validator's stake delegators. The rate can be any value in the range `0 - 100%`. At registration all validators have commission set to a default rate specified by the Autonity network's genesis configuration. (See Reference [Genesis, `delegationRate`](/reference/genesis/#configautonity-object).) After registration the validator can modify its commission rate - see [Validator commission rate change](/concepts/validator/#validator-commission-rate-change) on this page.
+-  The validator's commission rate on delegated stake. Commission is a percentage amount deducted by a validator from staking rewards before rewards are distributed to the validator's stake delegators. The rate can be any value in the range `0 - 100%`. At registration all validators have commission set to a default rate specified by the Autonity network's genesis configuration. (See Reference [Genesis, `delegationRate`](/reference/genesis/#configautonity-object).) After registration the validator can modify its commission rate - see [Validator commission rate change](/concepts/validator/#validator-commission-rate-change) on this page.
+
+Slashing reward revenue potential is determined by the validator reporting a proven fault committed by other validators while a member of the consensus committee. This is driven by:
```

Comment:
I do not understand this sentence, is it trying to explain what are the advantages for a misbehaviour reporter? I think the advantages for reporting a faults are only that you get the rewards of the faulty validator

Reply:
Hmm. I've simplified this to:

>Slashing rewards can be earned by a validator for reporting a slashable fault committed by another committee member per the [accountability and fault detection](/concepts/accountability/) protocol. As reward the _reporting validator_ is awarded the staking rewards the _offending validator_ would have earned for the epoch.

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

Reply:
much clearer 👍🏻